lib/mpl_toolkits/axisartist/grid_finder.py

```diff
@@ -153,12 +153,13 @@ def _get_raw_grid_lines(self,
 
 
     def _clip_grid_lines_and_find_ticks(self, lines, values, levs, bb):
-        gi = dict()
-        gi["values"] = []
-        gi["levels"] = []
-        gi["tick_levels"] = dict(left=[], bottom=[], right=[], top=[])
-        gi["tick_locs"] = dict(left=[], bottom=[], right=[], top=[])
-        gi["lines"] = []
+        gi = {
+            "values": [],
+            "levels": [],
+            "tick_levels": dict(left=[], bottom=[], right=[], top=[]),
+            "tick_locs": dict(left=[], bottom=[], right=[], top=[]),
+            "lines": [],
+        }
```

Comment:
This is OK, but I have a mild style preference for using `dict()` whenever possible:
```python
gi = dict(values=[],
          levels=[],
          tick_levels=dict(left=[], bottom=[], right=[], top=[]),
          #etc.
         )
```